server/api/users/users.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { RequestHandler } from 'express';
+import _ from 'lodash';
+
+import { User } from 'server/models';
+
+type ClientUser = Pick<User, 'id' | 'discord_user_id'>
+
+export const getCurrentUser: RequestHandler<void, ClientUser>  = (req, res) => {
+  // eslint-disable-next-line  @typescript-eslint/no-non-null-assertion
+  const filteredUser = _.pick(req.user!, [
+    'id',
+    'discord_user_id',
+  ]);
```

Comment:
We do this filtering so that if we add additional fields to the User model, we aren't including it in the client.  The client in general should know as little as possible in order to prevent unintentional data leakage.